views/index.html

```diff
@@ -0,0 +1,93 @@
+<html>
+<head>
+    <link href="/static/main.css" rel="stylesheet">
+</head>
+<body class="govuk-template__body">
+<header class="govuk-header " role="banner" data-module="header">
+    <div class="govuk-header__container govuk-width-container tna-header">
+
+        <div class="govuk-header__tna-logo">
+            <a href="/" class="govuk-header__tna_link govuk-header__tna_link--homepage">
+                <img src="/static/tna-horizontal-white-logo.svg"
+                     class="govuk-header__tna-logo-image" alt="TNA horizontal logo"/>
+            </a>
+        </div>
+
+        <div class="govuk-header__tna-content">
+            <a href="#" class="govuk-header__link govuk-header__link--service-name">
+                Transfer Digital Records
+            </a>
+        </div>
+    </div>
+</header>
+
+<div class="govuk-width-container">
+
+    <div class="govuk-phase-banner">
+        <p class="govuk-phase-banner__content">
+            <strong class="govuk-tag govuk-phase-banner__content__tag">BETA</strong>
+            <span class="govuk-phase-banner__text">This is a new service – your feedback will help us to improve it. Please
+                <a class="govuk-link" target="_blank" rel="noreferrer noopener" href="/contact">get in touch (opens in new tab).</a>
```

Comment:
Internal links on this page won't work because the service is down.

I think it's worth talking to Pia about whether we should just get rid of the links in the header and footer. I've pretty sure seen this elsewhere, where the service unavailable page has the same branding as the main site but doesn't have as many options because they wouldn't work.

Reply:
Good point. I've put a comment on the ticket and I'll mention it at next standup.

Reply:
Pia has updated the design to remove the links. I've changed the index file to match the new design.